packages/test/test-end-to-end-tests/src/test/directoryEndToEndTests.spec.ts

```diff
@@ -1209,6 +1209,30 @@ describeNoCompat("SharedDirectory orderSequentially", (getTestObjectProvider) =>
 		assert.equal(changedEventData[1].previousValue, undefined);
 	});
 
+	it("Should rollback deleted subdirectories with the original order", () => {
```

Comment:
Also add a test where the original creation is ack'ed.

Reply:
Added it in a separate test suite